src/r5py/r5/speed_config.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, **kwargs):
             See https://github.com/conveyal/r5/blob/v6.6/src/main/java/com/conveyal/r5/point_to_point/builder/SpeedConfig.java#L10
         """
         super().__init__()
-        kwargs = {util.snake_to_camel_case(key): value for key, value in kwargs}
```

Comment:
Looks good, but as a side note: Isn't this convention now going against what we discussed earlier here?:
https://github.com/r5py/r5py/pull/61#discussion_r871329494

Reply:
It’s a local import, so it goes with my understanding of the convention (classes and/or functions from within the module), but yes, it’s of course worth a discussion. Should it then be `r5py.util.snake_to_camel_case()`?

Reply:
hm, thinking about it, I would not want to import `r5py.util`, as I’m afraid an existing installation might override a local directory (but that might not be true, have to research a bit on that). Can we move this into a separate issue? It would be nice to define this convention (and related ones) more systematically, explore the implications (global import will call `__init__.py` first), and apply them throughout. In general, I have the feeling that in some places (e.g. `jvm.py`), my code relies too much on side effects (see also the `noqa: F401` comments). We should maybe have a `start_jvm_if_its_not_yet_running()` function, for instance.

Reply:
@christophfink : Sounds good, I see you have already created an issue for this in #95 👍🏻 Let's continue there. It would indeed be good to document / agree on the used conventions, because it will also help in the future when maintaining the package and when possible new contributors come in.